cirq/protocols/decompose.py

```diff
@@ -115,6 +102,10 @@ def decompose(
                                       Union[None,
                                             NotImplementedType,
                                             'cirq.OP_TREE']] = None,
+    fallback_decomposer: Callable[['cirq.Operation'],
```

Comment:
Think we should define a custom type that has this signature since this is used in multiple places

Reply:
Acknowledged. For now I think it's okay to just say it so that people don't have to deref when reading the docs.